kong/plugins/oauth2/api.lua

```diff
@@ -53,7 +55,7 @@ return {
           return endpoints.handle_error(err_t)
         end
         if not consumer then
-          return responses.send_HTTP_NOT_FOUND()
+          return kong.response.exit(HTTP_NOT_FOUND)
```

Comment:
In order to be completely equivalent to the response above, this should be

```
kong.response.exit(HTTP_NOT_FOUND, { message = "Not Found" })
```

Same in the change below.